code/game/machinery/computer/camera.dm

```diff
@@ -46,6 +46,12 @@
 			M.unset_machine() //to properly reset the view of the users if the console is deleted.
 	return ..()
 
+/obj/machinery/computer/security/proc/isFarAway(var/atom/C)
+	var/turf/CONSOLETURF = get_turf(C)
+	var/turf/T = get_turf(src)
+	if((is_away_level(T.z) || is_away_level(CONSOLETURF.z)) && !atoms_share_level(T, CONSOLETURF)) //can only recieve away mission cameras on away missions

```

Comment:
It's funny, but video cameras being able to transmit through the gateway is actually a bug. Since the camera is actually *inside* the videocam object itself, its `z` var is set to 0, so this will not `return` like it should do.